codespell.py

```diff
@@ -333,10 +339,15 @@ def main():
     prev = ""
     bc = 0
     for x in bad_words:
```

Comment:
Possibly improve readability, what are x[0], x[1], x[2]? Use explicit names, guessing here but something like the following will be more readable: 
```
for misspelled_word, optional_replacement, number_of_times in bad_words:
...
```